dev/doc-import-tool/src/extract/extract.ts

```diff
@@ -62,6 +63,13 @@ class TocContentExtractor implements ContentExtractor {
 export async function extract (contents: string, spec: FileSpec, headerRoot?: AnyNode): Promise<ExtractedFile> {
   const extractor = new TocContentExtractor(spec)
   const doc = parseDocument(contents)
+
+  // We do not support headers > 3 so
+  // Traverse all Document's childrent and replace all h4-h6 with paragraphs
+  findAll((n) => ['h4', 'h5', 'h6'].includes(n.tagName), doc.childNodes).forEach((node) => {
+    node.name = 'p'
```

Comment:
maybe add bold mark to highlight ex headings?

Reply:
From what we saw so far the actual usage of these headers was only to automatically enumerate them so most likely they will be converting them to lists where needed. Not sure if bold will be helpful for this scenario.